### ISSUE TITLE
Prevent component from reloading when re-arranging

### ISSUE THREAD

**Issue by dtbmobile** [mathuo/dockview#162]
Hello and thanks for this really great project,

can you tell me if there is a way to not reload the panel-content when rearranging its position? In certain situations it does, in others not:

For example:
I have a Dockview component including one panel. I open another one with openPanel in the direction "right" - Only the new dockview reloads, which is as expected. When I now open another panel with openPanel in the direction "below" the new panel AND the one before (the second) reload.

I hope I described it understandable - let me know if not, I am happy for any help.

Best regards

**Comment by mathuo** [comment]
Hey - sorry for the late reply.

By reloads do you mean the web page is repainting the content or something more specific to Reacts rendering? If possible an example could be very useful to see this in action. I was having a quick play in this sandbox but maybe you could adjust it to show your problem? https://codesandbox.io/s/inspiring-shadow-eoybz1?file=/src/App.tsx

**Comment by jogibear9988** [comment]
I've had a similar problem in dock-spawn-ts (https://node-projects.github.io/dock-spawn-ts/)
We fixed this, by keeping the content of the panel always in the dome, and positioning it absolute to the position where the dockwindow is.

**Comment by dtbmobile** [comment]
Hello, thanks for your replies!
I actually found out my problem is **not** related to dockview. I use the dockview to display multiple **iframes** - so when I investigated further by trying the same simple div - which is displaying its content after 5 seconds (setTimeout) - I get different results when using:
- just the div
- an iframe with srcDoc attribute
- an iframe with src attribute

If it is just the div or an iframe with the srcDoc attribute it worked perfectly fine! But an **iframe with the src attribute** results in this:
- I have one initial Panel
- I add another panel to the right -> only the new Panel reloads (darkgrey background means it is reloading)
- I add another panel below the second Panel -> second and third panel reload (this is my problem)

Thats how it looks:
![CPT2212131155-1517x865](https://user-images.githubusercontent.com/84180040/207299448-587fc75f-8638-4c4e-a5b3-b936bb0b119c.gif)

With divs and iframe with srcDoc attribute it behaves like this (as expected):
![CPT2212131200-1519x865](https://user-images.githubusercontent.com/84180040/207300673-f193cfbf-f5d8-49e8-8bad-eedac1e15109.gif)

So thanks for trying to help me here and again for the great dockview solution - I will investigate iframe behaviours instead :)

Best regards
